src/interfaces/chain.h

```diff
@@ -109,6 +110,10 @@ class Chain
 
         //! Check if transaction will be final given chain height current time.
         virtual bool checkFinalTx(const CTransaction& tx) = 0;
+
+        //! Add transaction to memory pool if the transaction fee is below the
+        //! amount specified by absurd_fee (as a safeguard). */
```

Comment:
in commit d02b34c8a8bd446c9620fe626b437:

The `absurd_fee` is only one case this could return false, but should never happen. There might be other cases such as descendant limits?